exe/manageiq-cross_repo

```diff
@@ -95,7 +95,10 @@ opts = Optimist.options do
   EOS
 end
 
+puts opts[:repos].inspect
 opts[:repos] = opts[:repos].flatten.flat_map { |repo| repo.split(",").map(&:strip) }
+puts opts[:repos].inspect
+exit
```

Comment:
I can't believe no one caught this debug code sneaking in...

Anyway, deleting, will push a fix shortly...

Reply:
ooh. I saw the WIP and assumed you had it in there for the short term.
I felt like mentioning it was trolling the obvious.

sorry.

Reply:
No, more like surprised with at least two people looking at it, no one made a mention.  Not your fault, it is mine, just surprising and mildly humorous/disappointing (for me).